splunk_eventgen/eventgen_nameko_controller.py

```diff
@@ -481,6 +482,20 @@ def process_server_confs(self):
         self.server_confs = {}
         return dump_value
 
+    def process_server_volumes(self):
+        current_server_vhosts = self.get_current_server_vhosts()
+        if current_server_vhosts:
+            # Try for 15 iterations to get results from current server vhosts
+            for i in range(15):
```

Comment:
why we hard coding this?  Seems like this should be a param somewhere.  Approving, but should probably make a ticket or issue to track replacing it, or add a #TODO statement to mark that this should be a param.

Reply:
I copied this code from another block, so I'm not sure :) I'll make a ticket for it.